src/components/Home/Home.scss

```diff
@@ -2,12 +2,9 @@
 
 .home_page {
 	&__container {
-		position: fixed;
 		overflow: auto;
-		height: 100%;
-		width: 100%;
-		margin: auto;
-		text-align: center;
+		height: 100vh;
```

Comment:
This will create an issue in mobile browsers. I will tell you the fix in today's meeting.